chainercv/experimental/links/model/fcis/fcis_resnet101.py

```diff
@@ -0,0 +1,234 @@
+from __future__ import division
+
+import chainer
+import chainer.functions as F
+import chainer.links as L
+import numpy as np
+
+from chainercv.experimental.links.model.fcis import FCIS
+from chainercv.functions import psroi_pooling_2d
+from chainercv.links import Conv2DBNActiv
+from chainercv.links.model.faster_rcnn.region_proposal_network import \
+    RegionProposalNetwork
+from chainercv.links.model.faster_rcnn.utils.loc2bbox import loc2bbox
+from chainercv.links.model.resnet.resblock import ResBlock
+from chainercv.utils import download_model
+
+
+class FCISResNet101(FCIS):
+
+    _models = {
+        'sbd': {
+            'n_fg_class': 20,
```

Comment:
Is setting the name of the pretrained weight `voc` inappropriate?

Reply:
VOC images with SBD annotations, so I prefer SBD.
Also, the model is trained with SBD training images, which differs from VOC train images.